templates/profile/info.yml.twig

```diff
@@ -0,0 +1,26 @@
+name: '{{ profile }}'
+type: 'profile'
+description: '{{ description }}'
+core: '8.x'
+distribution:
```

Comment:
I think it's safe to say that if someone is extending Lightning, they expect the extending profile to a distribution too.

My experience is that (with the core base profile patch) if a codebase has exactly one distribution, that distribution will always get installed, regardless of whether a profile exists that lists that distributions profile as a base profile. So I think this _needs_ to be there. (Without the core patch, I think you should get a TooManyDistributionsException.

Let's confirm/discuss further.